bundles/org.openhab.binding.shelly/src/test/java/org/openhab/binding/shelly/internal/discovery/ShellyThingCreatorTest.java

```diff
@@ -39,6 +39,12 @@ public class ShellyThingCreatorTest {
 
     private static final String DEVICE_ID = "000000000000";
 
+    @Test
+    void printSupportedThingTypes() {
+        System.out.println("Supported Things:");
+        SUPPORTED_THING_TYPES.stream().map(ThingTypeUID::toString).sorted().forEach(System.out::println);
+    }
+
```

Comment:
This was provided only for debugging purposes, rather than a test (no assertions, console output only).